launcher/plans/translator_nest_to_tvb_on_local.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<settings>
+    <co_simulation_action_plan>
+        <title>Test the translator Nest to TVB</title>
+        <description>Test the translator Nest to TVB</description>
+        <parameters>
+            <par_000>
+                <par_name>CO_SIM_XML_ACTIONS_DIR</par_name>
+                <par_value>${CO_SIM_TVB_NEST}/launcher/actions</par_value>
+            </par_000>
+            <par_001>
+                <par_name>CO_SIM_EXECUTION_ENVIRONMENT</par_name>
+                <par_value>Local</par_value>
+            </par_001>
+        </parameters>
+        <variables>
+            <!-- The variables under this section, will be created dynamically and held on a python dictionary
+                 NOTE: the NAME of the variable is the dictionary KEY to have access to its value
+            -->
+            <var_000>
+                <var_name>CO_SIM_DELAY</var_name>
+                <var_value>100</var_value>
+            </var_000>
+            <var_001>
+                <var_name>CO_SIM_RESULTS_DIR</var_name>
+                <!-- NOTE:  CO_SIM_RUNTIME_RESULTS_DIR will be
+                            generated on run-time by the configuration manager
```

Comment:
The paths to output directories (results, logs, etc.) can be provided to configurations manager in 'plan' files or we might can discuss the best way to provide this information to configurations manager.